modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureListView.testcase

```diff
@@ -34,188 +38,172 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-131206. Verify if its possible download envelopes by Envelope Description "
+	@description = "LPS-131206. Verify if its possible download envelopes by Envelope Description "
 	@priority = 3
 	test CanDownloadEnvelopeByEnvelopeDescription {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		DigitalSignature.createEnvelope(
-			addEmail = "test@liferay.com",
-			addFullName = "Recipient Full Name",
-			addMessage = "Email Message",
-			addName = "Test",
-			addSubject = "Email Subject",
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf");
-
-		Button.click(button = "Send");
-
-		Alert.viewSuccessMessageText(successMessage = "Your envelope was created successfully.");
-
-		Navigator.openURL();
+		task ("Given a digital envelope is created") {
+			DigitalSignature.createEnvelope(
+				addEmail = "test@liferay.com",
+				addFullName = "Recipient Full Name",
+				addMessage = "Email Message",
+				addName = "Test",
+				addSubject = "Email Subject",
+				dmDocumentFile = "Alice's Adventures in Wonderland.pdf");
+
+			Button.click(button = "Send");
+		}
 
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Digital Signature");
+		task ("Then a message appears that the envelope was successfully created") {
+			Alert.viewSuccessMessageText(successMessage = "Your envelope was created successfully.");
+		}
 
-		var envelopeName = "Test";
+		task ("Then the digital signature can be downloaded") {
+			Navigator.openURL();
 
-		Click(
-			envelopeName = ${envelopeName},
-			locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_ENVELOPE_NAME");
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Digital Signature");
 
-		AssertElementPresent(locator1 = "DigitalSignatureListView#DIGITAL_SIGNATURE_DOWNLOAD_BUTTON");
+			DigitalSignature.enabledDigitalSignatureDownload(documentName = "Test");
```

Comment:
This macro `DigitalSignature.enabledDigitalSignatureDownload` you can put in a task: `"Then the digital signature's envelope  can be downloaded"`